flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientTest.java

```diff
@@ -151,6 +152,33 @@ public void write(int b) throws IOException {
 		}
 	}
 
+	@Test
+	public void testInvalidateValue() throws Exception {
+		Executor executor = mock(Executor.class);
+		doThrow(new ValidationException("Property 'parallelism' must be a integer value but was: 10a"))
+			.when(executor).setSessionProperty(any(), any(), any());
+		InputStream inputStream = new ByteArrayInputStream("set execution.parallelism = 10a;\n".getBytes());
+		// don't care about the output
+		OutputStream outputStream = new OutputStream() {
```

Comment:
Don't we have utilities already somewhere?

Reply:
we can use {{MockOutputStream}} here after its access modifiers is changed from `private` to `public`.